### PR TITLE
WBK Lightsabers & Force Commando Blade Compat

### PR DESCRIPTION
**Pull request by liamcannon**
### Description
Adds Compat for WBK Lightsabers & Force Commando blade

### Changes
- Adds new patch for WBK Lightsaber & Force 
  - Mine Detector compat for WBK_KatarnCloneKnife

### CODE REVIEW

**Comment by liamcannon** [comment]
I think the current SUBADDON naming schemes could prove problematic. I went with MDF_WBKLightsaber for this one.

File: addons/Compats/WBKLightsaber/script_component.hpp
Comment (DartRuffian) [inline addons/Compats/WBKLightsaber/script_component.hpp:1]: Final compat addon name is PREFIX_Compats_SUBCOMPONENT.
Prefix doesn't need to be included in subcomponent name.

Comment (liamcannon) [reply addons/Compats/WBKLightsaber/script_component.hpp:1]: I misread the macros

**Comment by DartRuffian** [comment]
> I think the current SUBADDON naming schemes could prove problematic. I went with MDF_WBKLightsaber for this one.

Could just shorten the compat names to be `MDF_WBKLSF` (in this case). The sub addon names being `Prefix_Component_Patch` was just from how CBA's macros were defined, not necessarily something I intentionally chose

**Comment by DartRuffian** [comment]
Looks good to me

**Pull request merged**